packages/x-components/src/components/__tests__/snippet-callbacks.spec.ts

```diff
@@ -21,71 +23,89 @@ function renderSnippetCallbacks({
 }
 
 describe('testing SnippetCallbacks component', () => {
```

Comment:
Revert the changes here and skip these tests, they'll be fixed in the snippet callbacks task

```suggestion
describe.skip('testing SnippetCallbacks component', () => {
```